twisted/spread/banana.py

```diff
@@ -310,7 +311,7 @@ def sendEncoded(self, obj):
 
         @return: L{None}
         """
-        io = cStringIO.StringIO()
+        io = BytesIO()
         self._encode(obj, io.write)
```

Comment:
I think name 'io' should be changed to something else, may cause problems because of the core "io" module